### ISSUE TITLE
Untamable Komodo dragons

### ISSUE THREAD

**Issue by TwilightAce** [Alex-the-666/AlexsMobs#2136]
Komodo Dragons are not tamable either by right-click or just throwing it on the ground in 1.20.1 version 1.22.9. The Komodo dragon is not eating the rotten flesh by right-click and I've literally thrown over a thousand pieces of rotten flesh on the ground that it has eaten still it has yet to be tamed

**Comment by pxnce** [comment]
I am also having this same issue.

**Comment by AlexprojetMDC** [comment]
Yeah Bro the same issue happens to me. Lucky it's not only me. I discovered It today and I thought It was my computer.

**Comment by CrimsonCrips** [comment]
Made a pull request for this, that fixes it

**Comment by MAGLANGEL98** [comment]
I have the same issue in version 1.20.1,  I can't tame the Komodo dragon either.

**Comment by Aezurian** [comment]
Did the fix with ChatGPT
It's Datapack by the way.

https://drive.google.com/file/d/1g4N-yc_GaFVbMsX9nauhDlBjjajWMQAR/view?usp=sharing
